classes/YARPP_Core.php

```diff
@@ -216,21 +213,6 @@ public function get_option($option = null) {
 
 		return $current;
 	}
-
```

Comment:
removed YARPP Pro tab in settings. Everything in the YARPP plugin works without neeinmg to be unlocked. "Plugins may not contain functionality that is restricted or locked, only to be made available by payment or upgrade." see https://developer.wordpress.org/plugins/wordpress-org/detailed-plugin-guidelines/#5-trialware-is-not-permitted)